### DISCUSSION TITLE
Table: Custom boolean, dropdown, and file size filters

### DISCUSSION THREAD

**Discussion by HaveSpacesuit** [iTwin/iTwinUI-react#691] [Show and tell]
I built some custom filters that are fairly easy to plug into the iTwinUI-react table. I added a dropdown filter, a column-values dropdown filter, a boolean filter, and a file size filter.

![image](https://user-images.githubusercontent.com/15036346/170157615-3ab9d964-cc39-4f94-ae83-56912dc28933.png)

https://codesandbox.io/s/itwinui-react-minimal-example-forked-ebgxns?file=/src/App.tsx

**Comment by bentleyvk** [comment]
Good job! It is nice to hear that it was easy for you to add custom filters.
